pages/glossary/changes-in-content.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Changes in content
+key: changes-in-content
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - Audio output
+  - CSS styling
+  - DOM tree
+---
+
+A _change in the content_ of a [web page][] caused by an event occurs when, by comparing the [web page][] before and after the event firing, at least one of the following occurs:
+
+- **visible changes:** the rendered pixels change in any part of the document that is currently within the [viewport][] or that can be brought into the [viewport][] via scrolling; or
+- **accessibility tree changes:** any state, property or event of a node representing an [accessible object][] of the [accessibility tree][] changes, or any node is inserted in, or removed from the [accessibility tree][]; or
+- **audible changes:** the audio reproduced by the [web page][] changes.
```

Comment:
I have completely missed that "render" was also used for audio in W3C lingo 😥 
I'm backing up on previous comment, "render" is the correct word to use here. Sorry about that 😓

Reply:
No worries!